### PR TITLE
Improve text in starter guide

### PR DESCRIPTION
**Pull request by nop33**
1. Improved readability of the _Requirements_ section by adding a footnote, fixing indentation, fixing typos, and updating the text
2. Improved readability of the _Start your node_ section by fixing indentation, creating a code block for easy copy+paste of command, and adding emoji for fun

Visual comparison:

| Before  | After |
| ------------- | ------------- |
| image | image  |

### CODE REVIEW

File: src/Starter-Guide---How-to-Launch-your-node.md [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 2. (Windows only) Install [Windows Terminal](https://www.microsoft.com/p/windows-terminal/9n0dx20hk701)
-3. In your home folder:
-   - Windows: `C:\Users\<your-username`
-   - MacOS: `/Users/<your-username`
-   - Linux: /home/<your-username>
- Create a folder called **.alephium** and
- Make sure to show the hidden folders:
- - [on Windows](https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5)
- - [on MacOS](https://www.pcmag.com/how-to/how-to-access-your-macs-hidden-files)
+3. In your home folder[^1]:
```
Comment (polarker) [inline src/Starter-Guide---How-to-Launch-your-node.md:7]: Very sweet footnote ❤️

File: src/Starter-Guide---How-to-Launch-your-node.md [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
+      - [on Windows](https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5)
+      - [on MacOS](https://www.pcmag.com/how-to/how-to-access-your-macs-hidden-files)
 4. (Windows only): You will need to add one `Environment variables`
-  * For this you can follow: [This guide](https://java.com/en/download/help/path.html) and follow the `Windows` section
-  * You can add the two following variables in your user's variables:
-      - Click `New`
-      - In the variable name type `HOME`
-      - In the variable value: `C:\Users\<your-user-name>`
+   - To do this you can follow the `Windows` section of [this guide](https://java.com/en/download/help/path.html).
+   - Add the following two values in your user's variables:
```
Comment (tdroxler) [inline src/Starter-Guide---How-to-Launch-your-node.md:14]: can you fix the `two`? now there's only one variable (we use to have 2)

**Review by polarker — APPROVED** [review]
LGTM

File: src/Starter-Guide---How-to-Launch-your-node.md (hunk 1 above)
Comment (tdroxler) [inline src/Starter-Guide---How-to-Launch-your-node.md:7]: nice, I never did a footnote in md

**Review by tdroxler — APPROVED** [review]
one comment, otherwise LGTM

**Comment by nop33** [comment]
@polarker @tdroxler thanks for your feedback!

I've added the suggestions and moreover I:
1. added a title to the document and changed the hierarchy of the titles of the document's sections from H1 to H2
2. highlighted the explorer link by adding it inside a blockquote

Latest preview:

image

File: src/Starter-Guide---How-to-Launch-your-node.md (hunk 2 above)
Comment (nop33) [reply src/Starter-Guide---How-to-Launch-your-node.md:14]: done

**Comment by polarker** [comment]
Feel free to merge now, thanks !

**Pull request merged**